defaults/main.yml

```diff
@@ -112,6 +112,15 @@ telemetry_tls_privatekey:
 telemetry_tls_certificate:
 telemetry_tls_ca_certificate:
 
+# JWT configuration
+jwt_key_path: /etc/hub/jwt
+jwt_privatekey:
```

Comment:
would it make sense to define some default name for both private and public key here so that a user of the role does not need to come up with a name?

Reply:
currently the name is passed via ansible_vars. we can also make it hard coded if you prefere.